packages/sado-connect/src/main.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { SadoConnectKit } from "./index.ts";
-import { AddressProvider } from "./providers/AddressContext.tsx";
+import { SadoConnectProvider } from "./providers/SadoContext.tsx";
+import "./style.css";
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <AddressProvider>
-      <SadoConnectKit />
-    </AddressProvider>
+    <SadoConnectProvider>
+      <SadoConnectKit
+        onViewWallet={() => console.log("You can provide a callback here.")}
```

Comment:
instead of using console.log, let's add comment to the component to describe the props

Reply:
we can use jsdoc style

Reply:
Can you link me to an example?

Reply:
https://github.com/BirthdayResearch/jellyfishsdk/blob/main/packages/whale-api-client/src/api/governance.ts#L7

Reply:
fyi, in vscode we can type `/**` and it will have jsdoc auto complete option

Reply:
added jsdoc